src/test/org/apache/nutch/api/AbstractNutchAPITestBase.java

```diff
@@ -160,6 +177,37 @@ private void initializeSSLProperties() {
     conf.set("restapi.auth.ssl.keypass", "password");
   }
 
+  private void loadCertificate() throws CertificateException, NoSuchAlgorithmException, KeyStoreException, IOException {
+    KeyStore ks = KeyStore.getInstance(KeyStore.getDefaultType());
+    char[] password = "testpassword".toCharArray();
+    String testTrustKeyStorePath = "src/test/testTrustKeyStore";
+    System.setProperty("javax.net.ssl.trustStore" , testTrustKeyStorePath);
+
+    File file = new File(testTrustKeyStorePath);
+    InputStream localCertIn = new FileInputStream(file);
+
+    ks.load(localCertIn, password);
+    localCertIn.close();
+
+    if (ks.containsAlias("nutch")) {
+      return;
+    }
+
+    InputStream certIn = new FileInputStream("src/test/nutch.cer");
```

Comment:
Yes. Whenever we load resources.... it should most likely always be done this way.